beluga/include/beluga/sensor/data/point_cloud.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2023 Ekumen, Inc.
```

Comment:
@pvela2017 nit: 
```suggestion
// Copyright 2024 Ekumen, Inc.
```
same elsewhere.

Reply:
Changed in commit 01d32b0098690a2ea9c25d1ffa1ca3e8311baafe